src/conf.py

```diff
@@ -101,14 +103,14 @@
 #keep_warnings = False
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = False
+todo_include_todos = True
 
 
 # -- Options for HTML output ----------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
Please revert this line

Reply:
I have no strong opinion on the rtd theme, but I really think alabaster does a very poor in making obvious things obvious.

Can we find a compromise that keeps a better user experience but is not RTD?

Reply:
I am not in love with Alabaster, but I do think it is cleaner than RTD. Probably the best thing to do ultimately is to make a theme configuration that matches our landing page (conda-forge.org), but I am definitely not asking you to do that here :)